crates/ledger-types/src/l1vs.rs

```diff
@@ -41,9 +47,17 @@ pub trait IL1ViewState {
     /// manifest we've accepted.
     fn set_asm_recorded_epoch(&mut self, epoch: EpochCommitment);
 
+    /// Gets the l1 references MMR.
+    ///
+    /// This is for the accounts to make references about L1 blocks.
+    fn l1_references(&self) -> &Mmr64;
```

Comment:
This name is unclear, it should be called something more precise like `asm_manifest_mmr`.

Also, group it with `append_manifest` so that it's clear the two fns are related.